src/mbgl/geometry/pbf_geometry.hpp

```diff
@@ -8,10 +8,10 @@
 
 namespace mbgl {
 
-class Geometry : private util::noncopyable {
+class PBFGeometry : private util::noncopyable {
 
 public:
-    inline explicit Geometry(pbf& data);
+    inline explicit PBFGeometry(pbf data);
```

Comment:
Accident, I think — was going to take a review back through and improve passing by reference where applicable.